### PR TITLE
API Console improvements

### PR DESCRIPTION
**Pull request by hihuz**
I'd like to do a few small updates to the API console:

* Update SDK to latest 0.0.16 version: 8f5a8b209f7c4af55217a9416e6eef8b8f683076
* Lint an auto-generated file that was not formatted properly: c6520aaa55d571a9e017d455a11496df8a064378
* Remove some legacy ts-ignore comments (possible thanks to the updated SDK): dbf0d44dcd7cf869d7e907c344b18441b15ecc9b
* Use types from Kontist client Schema directly instead of defining our own: 0dc6bf6f800557d1c11e8d70b43a8f25d3c4355e

### CODE REVIEW

File: console-src/src/types/oAuthClient.tsx [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import * as Schema from "@kontist/client/dist/graphql/schema";
```
Comment (hihuz) [inline console-src/src/types/oAuthClient.tsx:1]: This works for now but shouldn't we expose these types in a more straightforward fashion in our SDK?

File: console-src/src/components/ClientForm/ClientForm.tsx [hunk 2]
```diff
@@ -66,17 +66,26 @@ type ClientFormProps = {
...
 
+type AuthorizedScopes =
+  | "OFFLINE"
+  | "ACCOUNTS"
+  | "USERS"
+  | "TRANSACTIONS"
+  | "TRANSFERS"
+  | "SUBSCRIPTIONS"
+  | "STATEMENTS";
+
```
Comment (johannespfeiffer) [inline console-src/src/components/ClientForm/ClientForm.tsx]: I think we can remove some duplicated code here with:
```
type AuthorizedScopes = Exclude<ScopeType, ScopeType.Clients | ScopeType.Admin>

type Scopes = {
  [index in AuthorizedScopes]: boolean;
}
```

Comment (hihuz) [reply console-src/src/components/ClientForm/ClientForm.tsx]: Thank you 🙏I was looking for exactly something like that 🙂 Didn't know about this `Exclude` yet.

I have updated this here: 449b560d42a4e603e3a92c8031a1c294f56832a2

File: console-src/src/types/oAuthClient.tsx (hunk 1 above)
Comment (johannespfeiffer) [reply console-src/src/types/oAuthClient.tsx:1]: Makes sense. Done in https://github.com/kontist/js-sdk/commit/931242f3dda6429682443d2d1fc3ee9466146a68. Should now work with
`import { Schema } from "@kontist/client";`

**Review by johannespfeiffer — APPROVED** [review]
LGTM 👍

File: console-src/src/types/oAuthClient.tsx (hunk 1 above)
Comment (hihuz) [reply console-src/src/types/oAuthClient.tsx:1]: I'll update it once we have this included in a release 🙂I'll merge the PR like this and will open another one when appropriate

**Pull request merged**